sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetRowConverter.scala

```diff
@@ -318,10 +318,32 @@ private[parquet] class ParquetRowConverter(
         new ParquetMapConverter(parquetType.asGroupType(), t, updater)
 
       case t: StructType =>
+        val wrappedUpdater = {
```

Comment:
@JoshRosen, no big deal at all but how about we put the JIRA ID somewhere in the comment?

Reply:
Good idea: I added a JIRA reference in https://github.com/apache/spark/pull/26993/commits/e6945e88a24d51551cba105b5e7e3825bc5e0a69